motr/client.c

```diff
@@ -862,6 +863,11 @@ void m0_op_fini(struct m0_op *op)
 		oc->oc_cb_fini(oc);
 	m0_mutex_fini(&op->op_priv_lock);
 
+	m0_tl_teardown(spti, &op->op_pending_tx, iter)
+		m0_free0(&iter);
+	spti_tlist_fini(&op->op_pending_tx);
+	m0_mutex_fini(&op->op_pending_tx_lock);
+
```

Comment:
This change uncovers a bug in m0_op_get() and breaks motr/ut/client.c:ut_test_m0_op_fini(). See https://github.com/Seagate/cortx-motr/pull/1599.